src/eval/tests.rs

```diff
@@ -22,7 +22,6 @@ fn parse(s: &str) -> Option<RichTerm> {
     grammar::TermParser::new()
         .parse_term(id, lexer::Lexer::new(s))
         .map(RichTerm::without_pos)
-        .map_err(|err| println!("{:?}", err))
```

Comment:
Is there a reason to remove that? Printing only happen when the test fail, IIRC. Those print statements are probably on purpose, to see the error when the test fail

Reply:
Ah okay! I ran the tests with `--nocapture` at one point to see some debug info and was surprised by how noisy it was so I just did a sweep to get rid of all the `println`s :sweat_smile: 

I didn't actually know that they get printed on test failure! That's very cool. I'll put them back in in that case.